releasenotes/notes/39521.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: release-notes/v2
+kind: bug-fix
+area: telemetry
+issue:
+  - https://github.com/istio/istio/issues/39468
+releaseNotes:
+  - |
+    **Fixed** a bug when specifying multi `accessLogging` in Telemetry, only last one is used.
```

Comment:
this needs to be much clearer, as within a single stanza, you could support multiple access logs and they would all be respected.